checkout.sh

```diff
@@ -7,6 +7,6 @@
 git clone https://github.com/mozilla-services/shavar-prod-lists.git || exit 1
 
 # This revision is taken from the original Cartfile.resolved
-(cd shavar-prod-lists && git checkout -q c938da47c4880a48ac40d535caff74dac1d4d77b)
+(cd shavar-prod-lists && git checkout -q 56e01bfb2ee84349289e33ea785dea0946fe0030)
```

Comment:
@say-yawn what do you think of this pinned commit? I'm not sure what to make of it .. it is on the 86.0 branch of the Shavar project, but not the latest on that branch.

Reply:
Yeah it would be nice to understand that.  That commit is the one used on firefox-ios too so we may need to update to a newer commit on that branch. 
I would like to note that I have not used a newer branch (87, or 88) because they don't have the normalized-lists folder with `base-fingerprinting-track.json` file, which is used both on firefox-ios and focus. It may be that we don't need it anymore? 
Thank you in advance @say-yawn!

Reply:
@say-yawn would it be better if we pinned to the last commit of the v86 branch? We get a few more updates then.

Reply:
Hi both, thanks for reaching out and asking about this. I need to manually generate the normalized lists for the fingerprinting lists. I can do that today. I recommend using the latest Desktop release branch (right now it would be 88 but will be 89 on June 1st) and its commit. I can generate the normalized fingerprinting list for both 88 and 89 versioned lists today. 

I do not recommend getting the latest versioned branch because I roll out new blocklists in Nightly (main branch) and Beta (Release - 1 version) to make sure the new tracking list is NOT causing major breakage. I also recommend updating the branch you are getting the lists from because the addition to the lists and removal from the lists will be available in the later versioned lists. If you would like to learn more on the list versioning and processes, checkout [this document](https://github.com/mozilla-services/shavar-prod-lists/#list-versioning-and-release-process).

Reply:
Thank you @say-yawn for the explanation and help here. Sounds good then to wait until June 1st and use the same version as desktop. I will open an issue to update also firefox-ios.